server/app/controllers/api/v1/api_users_controller.rb

```diff
@@ -10,38 +10,24 @@ def index
     @api_users = @api_users.where(mentee: true) if params[:mentee] == 'true'
   end
 
+
   # Post /api_users
   def create
     @api_user = ApiUser.create!(api_user_params)
-    json_response(@api_user, :created)
+    @auth_token = AuthenticateUser.new(@api_user.email, @api_user.password).call
+    # response = { message: Message.account_created, auth_token: auth_token, api_user: @api_user }
+    # json_response(response, :created)
```

Comment:
The return value of this method, it seems to have changed. Into what?

Reply:
if you create the user successfully the return value is just the auth token for a session under that user id. I have removed the commented out code